pages/checkoutPage.ts

```diff
@@ -0,0 +1,45 @@
+import {Locator, Page, expect} from "@playwright/test";
+
+export class CheckoutPage {
+    readonly page: Page;
+    readonly paymentDropdown: Locator;
+    readonly creditCardInput: Locator;
+    readonly expirationDateInput: Locator;
+    readonly cvvInput: Locator;
+    readonly cardHolderNameInput: Locator;
+    readonly confirmButton: Locator;
+    readonly paymentSuccessMessage: Locator;
+
+
+    constructor (page: Page) {
+    this.page = page;
+    this.paymentDropdown = page.getByTestId("payment-method");
+    this.creditCardInput = page.getByTestId("credit_card_number");
+    this.expirationDateInput = page.getByTestId("expiration_date");
+    this.cvvInput = page.getByTestId("cvv");
+    this.cardHolderNameInput = page.getByTestId("card_holder_name");
+    this.confirmButton = page.getByTestId("finish");
+    this.paymentSuccessMessage = page.getByTestId("payment-success-message");
+    }
+
+async selectPaymentMethod (): Promise<void> {
+    await this.paymentDropdown.selectOption("credit-card");
+}
+
+async fillPaymentForm (cardNumber: string, date: string, cvv: string, holderName: string): Promise<void> {
```

Comment:
I'd pass an object as a parameter


```suggestion
async fillPaymentForm (cardData: { cardNumber: string, date: string, cvv: string, holderName: string }): Promise<void> {
```
Destructuring
```
const {cardNumber, date, cvv, holderName} = cardData;
```
Function takes a lot of parameters, so it's easy to pass a wrong parameter